test/link/names2.txt

```diff
@@ -0,0 +1,77 @@
+;;; TOOL: run-wasm-link
+;;; FLAGS: --debug-names -r
+(module
+  (import "__extern" "missing" (func $import_func2))
+  (import "__extern" "baz" (func $import_func1))
```

Comment:
Sure, updated. Unfortunately, using `relocate_func_index` results in deviating test output.